metriq_gym/process.py

```diff
@@ -0,0 +1,156 @@
+"""Benchmark processing and calculation utilities."""
+import json
+import math
+import statistics
+
+from scipy.stats import binom
+
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit.providers import Job
+
+from metriq_gym.bench import BenchJobResult, BenchProvider
+
+def get_job(result: BenchJobResult) -> Job:
+    if result.provider == BenchProvider.IBMQ:
+        return QiskitRuntimeService().job(result.id)
+
+    raise ValueError(f"Cannot find provider {job.provider}.")
```

Comment:
This `raise` error (if hit) will fail as `job` is not defined.

Reply:
Fixed.